charts/sorry-cypress/templates/deployment-api.yml

```diff
@@ -48,6 +48,8 @@ spec:
         {{-  $defaultConnectionSecretName := include "sorry-cypress-helm.fullname" . | printf "%s-mongodbsecrets" }}
         {{-  $defaultConnectionSecretKey := "MONGODB_URI" }}
         {{- if .Values.mongodb.mongoSecretConnectionString.enableCustomSecret }}
```

Comment:
Why is this nested within the mongo if?

Reply:
something was wrong during a merge conflict, I guess